assets/training/finetune_acft_image/environments/acft_video_mmtracking/context/requirements.txt

```diff
@@ -0,0 +1,17 @@
+azureml-metrics[video]====0.0.31
+azureml-evaluate-mlflow=={{latest-pypi-version}}
+azureml-acft-accelerator=={{latest-pypi-version}}
+azureml-acft-common-components[image]=={{latest-pypi-version}}
```

Comment:
It looks like the pinning is in common-components for mmdetection environments. Could you please check.